### DISCUSSION TITLE
网站部署讨论

### DISCUSSION THREAD

**Discussion by JulyGlow** [sjt-2024/tgyz2602#1] [General]
就地开始

**Comment by lingbopro** [comment]
来咯

**Reply by Minemetero** [reply]
来了来了

**Comment by lingbopro** [comment]
接着我们在 https://github.com/lingbopro/lingbos-sussy-mod/discussions/1#discussioncomment-10294869 的结果继续

**Comment by lingbopro** [comment]
@Minemetero 这个网站的样式是直接用库还是自己写？

**Reply by Minemetero** [reply]
快速吧

**Reply by lingbopro** [reply]
？

**Reply by Minemetero** [reply]
我看他这个要求不高用标准库足以 :)

**Reply by Minemetero** [reply]
sober.js够满足了而且我可以不用

**Reply by lingbopro** [reply]
ok

**Reply by Minemetero** [reply]
要不这样，我看他这个肯定不止一个page，等他需求明确了我们分工

**Reply by lingbopro** [reply]
> 要不这样，我看他这个肯定不止一个page，等他需求明确了我们分工

那也没事

**Comment by lingbopro** [comment]
我先clone下来，把骨架做了

**Reply by lingbopro** [reply]
我这VSCode变慢了

**Reply by lingbopro** [reply]
![image](https://github.com/user-attachments/assets/0f581b97-5c59-43e8-8711-8568a3ab9a39)

...

**Reply by Minemetero** [reply]
行动真快，我都还没搞清楚他的需求呢

**Reply by Minemetero** [reply]
你不应该git clone 嘛

**Reply by Minemetero** [reply]
他没给我协作者

**Reply by lingbopro** [reply]
> 你不应该git clone 嘛

我确实clone的，但是在VSCode里操作的

**Reply by lingbopro** [reply]
![image](https://github.com/user-attachments/assets/ae2421ad-5c77-466c-ba76-1878a54ff979)

我真服了

**Reply by JulyGlow** [reply]
你敲下git-bash,或者拿加速器顶上

**Reply by JulyGlow** [reply]
@Minemetero 协作者给你了,怎么没通过?

**Reply by lingbopro** [reply]
魔幻，挂了加速器clone不下来，关了就行了

**Reply by JulyGlow** [reply]
err..挂的啥子加速器啊(我的Fast-github

**Reply by lingbopro** [reply]
> err..挂的啥子加速器啊(我的Fast-github

steam++

**Reply by JulyGlow** [reply]
fastgithub还有ssh代理呢

**Reply by Minemetero** [reply]
他用easy-github-hosts :)

**Reply by lingbopro** [reply]
> 他用easy-github-hosts :)

有道理，很好的机会啊（（（

**Comment by lingbopro** [comment]
那个，Sober库用CDN还是直接扔库里？

**Reply by JulyGlow** [reply]
可是github要挂加速器,我们班没谁会挂,这访问的问题...有待解决

**Reply by lingbopro** [reply]
> 可是github要挂加速器,我们班没谁会挂,这访问的问题...有待解决

github.io 貌似不挂也行吧

**Reply by JulyGlow** [reply]
> > 可是github要挂加速器,我们班没谁会挂,这访问的问题...有待解决
> 
> github.io 貌似不挂也行吧  

咳咳,我让我们班同学去看我的主页,都上不去

**Reply by lingbopro** [reply]
> > > 可是github要挂加速器,我们班没谁会挂,这访问的问题...有待解决
> > 
> > 
> > github.io 貌似不挂也行吧
> 
> 咳咳,我让我们班同学去看我的主页,都上不去

也许可以用镜像？

**Comment by lingbopro** [comment]
![image](https://github.com/user-attachments/assets/3bd8e61d-0e8b-418a-84e7-7aee992eb526)

Sober实现我issue了:)

**Reply by lingbopro** [reply]
@Minemetero Sober更新了，咱又来活了

**Comment by lingbopro** [comment]
![image](https://github.com/user-attachments/assets/fcb8a939-2b2d-4393-b018-1f53b3f3560f)

有了我做的扩展还是便捷

**Reply by Minemetero** [reply]
我的评价是。。。还是废话很多

**Comment by lingbopro** [comment]
我找到了一些 github.io 访问加速的方法

https://monkeywie.cn/2020/08/20/fast-github-page-with-cloudflare/
https://xiaowangye.org/posts/using-cloudflare-worker-proxy-github/

如果要需要域名的话可以去 <https://nic.eu.org> 申请一个

**Comment by lingbopro** [comment]
我们可以准备一个Actions工作流，在GitHub仓库提交时自动推送到Gitee Pages，这样直接访问Gitee Pages就很快了

**Reply by JulyGlow** [reply]
可是我没有gitee,没有手机号可以注册

**Reply by lingbopro** [reply]
我找到一篇文章：
https://blog.csdn.net/qq_35787254/article/details/120873489

看起来国内免费且好用的静态站点托管服务不多，还得选国外的，不过我没测访问速度

**Reply by lingbopro** [reply]
![Screenshot_20240811_091820.jpg](https://github.com/user-attachments/assets/963a39b2-3981-4109-a83a-f6643ccde224)

这是GitHub Pages的访问图，每个地区总有一个运营商的访问不通

**Comment by JulyGlow** [comment]
@lingbopro @Minemetero 
我把仓库名改了(方便访问),你们push时注意下

**Reply by JulyGlow** [reply]
呃,我又不得不改回来
sorry

**Comment by lingbopro** [comment]
关于 https://blog.csdn.net/qq_35787254/article/details/120873489 这篇文章里提到的几个平台：

1. GitHub Pages，不用说了，我贴一下访问图
  ![Screenshot_20240811_092230_com.microsoft.emmx.jpg](https://github.com/user-attachments/assets/ee428fa7-0f7f-477c-bfa4-77840c5c7ee7)
2. BitBucket，我上不去
3. aerobatic，域名都没了
4. GitLab，唯一一个彳亍的，请看VCR (苦了海南了)
  ![Screenshot_20240811_092627_com.microsoft.emmx.jpg](https://github.com/user-attachments/assets/ea8718e5-af68-41cc-9ef2-de4fe2371046)
5. surge.sh，我能上，但是请看VCR
  ![Screenshot_20240811_092450_com.microsoft.emmx.jpg](https://github.com/user-attachments/assets/4840ca40-9f88-4398-b463-2d5385554f6b)
6. Firebase Hosting，谷歌的，别想了
7. Netlify，看图：
  ![Screenshot_20240811_092208_com.microsoft.emmx.jpg](https://github.com/user-attachments/assets/a21af598-1c86-4af8-a86f-3b6910e37800)
8. now，我上不去

**Reply by Minemetero** [reply]
............

**Reply by Minemetero** [reply]
勤奋的开发团队

**Reply by JulyGlow** [reply]
咱看着是没问题,但是他们从来没上过github,所以他们刷新几次,还是无法访问(更别说看他们的错误码了,一堆internet盲,只会刷视频和喷)    
也不知道是不是我改了hosts文件,我关了加速器还能上github.io  
算了,到时候看,我能搞CDN就搞

**Reply by lingbopro** [reply]
目前来看，镜像的最佳选择是GitLab

**Reply by lingbopro** [reply]
我不改hosts裸连github.io很通畅

**Reply by kjmjh** [reply]
我这边对github很友好，大部分时间都能访问。所以我一般不需要加速。。。

**Reply by Minemetero** [reply]
死去的chat

**Reply by kjmjh** [reply]
😄

**Comment by lingbopro** [comment]
这图片位置和大小的问题要把我干废了

![image](https://github.com/user-attachments/assets/ecc40c21-20e8-4490-b11a-148c6df316c9)

**Comment by lingbopro** [comment]
目前我做了一点，要不push上来看看？

**Reply by JulyGlow** [reply]
让俺康康
(刚才fastgithub真的把我气死

**Reply by lingbopro** [reply]
我又push不上去了

**Reply by lingbopro** [reply]
ok了

git报`OpenSSL SSL_read: SSL_ERROR_SYSCALL, errno 0`的解决方法就是关了SSL校验

**Comment by JulyGlow** [comment]
设计的可以,但是我们的网站以后都要更新的,所以,麻烦你们加上八年级和九年级的选项卡,九年级的内容是NONE.我争取每次活动后更新下网站
我吃完午饭就找我们老师要图片,等我哈.
顺便教下我怎么改描述,怎么展示图片,非常感谢咯.

**Reply by lingbopro** [reply]
在`assets/js/data.js`里改，页面的信息都是从这个文件里的JSON信息动态生成的

**Reply by lingbopro** [reply]
话说，为什么要删README呢

**Reply by JulyGlow** [reply]
ah?要是加了多个图片呢?还有多个选项卡

**Reply by JulyGlow** [reply]
我终于看到了前端的强大

**Reply by lingbopro** [reply]
> ah?要是加了多个图片呢?还有多个选项卡

我设计的就是数组，`semesters`是学期，`pictures`是这个学期的图片列表

**Reply by Minemetero** [reply]
我还在Java的水深火热无法自拔

**Reply by JulyGlow** [reply]
咱这图片引用就直接写文件名啊?不要写路径吗(相对 or 绝对)?

**Reply by lingbopro** [reply]
> 咱这图片引用就直接写文件名啊?不要写路径吗(相对 or 绝对)?

是相对于img文件夹的路径，图片放这文件夹里，filename写文件名

**Reply by lingbopro** [reply]
话说，要不要搞个logo？

**Reply by JulyGlow** [reply]
logo,造什么好呢?我们班又没有logo.要不然我去造一个(之前美术课造过)

**Reply by JulyGlow** [reply]
硬是没找到一个合适的logo

**Reply by lingbopro** [reply]
> 硬是没找到一个合适的logo

那…自己做一个？

**Comment by lingbopro** [comment]
![Screenshot_20240811_133602.jpg](https://github.com/user-attachments/assets/fa8e8a47-d849-4438-85ef-f2989a78e6f9)

？

**Reply by JulyGlow** [reply]
err..好像没问题吧

**Reply by lingbopro** [reply]
所以这两个人是什情况

**Reply by JulyGlow** [reply]
可能他们是到处挖项目的

**Reply by lingbopro** [reply]
emmm

**Reply by JulyGlow** [reply]
@lingbopro 七年级选项卡左侧,我想加一个班级简介,内容我来写.行不?

**Reply by lingbopro** [reply]
行，那这么说我还得改

**Reply by JulyGlow** [reply]
到时候告诉我在哪里插入文本就行了 (真的非常感谢你们啊

**Reply by lingbopro** [reply]
![image](https://github.com/user-attachments/assets/dc6ad206-98b1-498c-b22f-d25b1d334888)

@sjt-2024 做成这种侧拉栏如何？

**Reply by JulyGlow** [reply]
嗯,非常可以

**Reply by JulyGlow** [reply]
底部要不要打上我们的账号,写上建造者?

**Reply by lingbopro** [reply]
> 底部要不要打上我们的账号,写上建造者?

我右上角放了一个信息按钮，可以放那，或者页脚也行

**Reply by lingbopro** [reply]
侧栏我push了

**Reply by Minemetero** [reply]
字体要换

**Reply by Minemetero** [reply]
布局要换。。。太粗糙了

**Reply by lingbopro** [reply]
> 字体要换

什

**Reply by lingbopro** [reply]
字体用系统默认挺好啊

**Reply by JulyGlow** [reply]
咋子个粗糙?font选什么?
我感觉建造者可以放在班级简介下面

**Comment by lingbopro** [comment]
目前这个布局看着还是不太舒服

我觉得可以把简介和图片分两个页面（可以还是放在一个html里）

**Reply by JulyGlow** [reply]
有道理,先造个首页

**Reply by lingbopro** [reply]
VSCode，再次启动！

**Reply by JulyGlow** [reply]
你们进度怎么样了?

**Reply by lingbopro** [reply]
分页面搞定，push上去了

**Reply by Minemetero** [reply]
我摆烂

**Reply by lingbopro** [reply]
> 我摆烂

你是一点更改没搞啊

**Reply by Minemetero** [reply]
我不知道你们要做啥我用户需求都没搞清楚，而且你在做我怕冲突

**Reply by lingbopro** [reply]
> 我不知道你们要做啥我用户需求都没搞清楚，而且你在做我怕冲突

我现在没做，你可以做了:D

**Reply by lingbopro** [reply]
> 我不知道你们要做啥我用户需求都没搞清楚，而且你在做我怕冲突

这用户需求挺清楚的啊

**Reply by Minemetero** [reply]
你做完了我来优化就行

**Reply by Minemetero** [reply]
你这不是做完了吗，还要做什么啊

**Reply by JulyGlow** [reply]
打开侧边栏,然后打开图片,有一部分会被侧边栏挡住,能优化下吗?
![image](https://github.com/user-attachments/assets/b10c8f46-be5f-4070-bda7-ea8c04015aa4)

**Reply by Minemetero** [reply]
我不知道我刚刚改了什么东东

**Reply by Minemetero** [reply]
> 打开侧边栏,然后打开图片,有一部分会被侧边栏挡住,能优化下吗? ![image](https://private-user-images.githubusercontent.com/174096511/356872686-b10c8f46-be5f-4070-bda7-ea8c04015aa4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzNzk0MzYsIm5iZiI6MTcyMzM3OTEzNiwicGF0aCI6Ii8xNzQwOTY1MTEvMzU2ODcyNjg2LWIxMGM4ZjQ2LWJlNWYtNDA3MC1iZGE3LWVhOGMwNDAxNWFhNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxMVQxMjI1MzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hM2VlMWFlMGJjYWQ4MjdjYmQ4N2ZiNzlkYTA1MjQ0ZmM3MzM3Mjc0NDliMDg3NTE2ZjA2ZWMwODYxOGRhYTc3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.0ZpOFvkKk7jAgwajAV-u9de_QhGnIE3EZ2X71VEpq8o)

你应该开issue的

**Comment by Minemetero** [comment]
那么接下来干什么。。。。

**Reply by lingbopro** [reply]
我不到啊

**Reply by lingbopro** [reply]
首先要等@sjt-2024 把图填上

**Reply by Minemetero** [reply]
我在做一些奇奇怪怪的东西，到时候放到dev上给你看看

**Reply by Minemetero** [reply]
搞完了你看看dev

**Reply by Minemetero** [reply]
导入本地看

**Reply by Minemetero** [reply]
我只做了那个功能其他的有没有影响还没测试

**Reply by lingbopro** [reply]
所以你删我注释干啥

**Reply by lingbopro** [reply]
另外加载页面完全不需要其它的效果，而且你搞的“下滑放大效果”是什么鬼

**Reply by Minemetero** [reply]
ohoh我又测试了一下压缩工具忘记了你的注释

**Reply by Minemetero** [reply]
> 另外加载页面完全不需要其它的效果，而且你搞的“下滑放大效果”是什么鬼

：）无聊搞得

**Reply by Minemetero** [reply]
本来就是dev

**Reply by Minemetero** [reply]
我想做出那个你windows更新时候的效果

**Reply by lingbopro** [reply]
> 我想做出那个你windows更新时候的效果

？

**Reply by Minemetero** [reply]
而且你加载画面一闪一闪的也不好看还不如直接延迟1秒再进入

**Reply by Minemetero** [reply]
而且你看加载界面既然做了干脆做好看点不辜负别人看我们一个片头

**Reply by lingbopro** [reply]
那干脆把加载页面删了吧

**Reply by Minemetero** [reply]
en?

**Reply by lingbopro** [reply]
我之前用edge的DevTools模拟了慢速网络，然后加载界面貌似没啥用处

**Reply by Minemetero** [reply]
oh

**Reply by lingbopro** [reply]
因为我们的CSS里有这么一条：

```css
:not(:defined) {
  display: none;
}
```

这一条指定未解析和未定义的元素不显示，而加载界面是在`<s-page>`里的，Sober未加载前`<s-page>`就处于未定义状态，所以也不会显示加载界面

**Reply by Minemetero** [reply]
![image_2024-08-12_120007288](https://github.com/user-attachments/assets/e0a44def-d129-493d-8b78-5275650791d9)
这样写你看咋样

**Reply by lingbopro** [reply]
你这是哪啊

**Reply by Minemetero** [reply]
我还没push

**Reply by Minemetero** [reply]
在更多信息下面

**Reply by Minemetero** [reply]
放的地方和放Github的地方一样

**Reply by lingbopro** [reply]
你这是在哪个分支

**Reply by lingbopro** [reply]
可以放GitHub上面，给我们名字加个链接到GotHub主页

**Reply by Minemetero** [reply]
你看要改嘛。。我push了

**Reply by lingbopro** [reply]
让我康康

**Reply by Minemetero** [reply]
好的

**Reply by Minemetero** [reply]
我push一下

**Reply by Minemetero** [reply]
草push错地方了

**Reply by Minemetero** [reply]
push到master了

**Reply by lingbopro** [reply]
你把这个做进master吧，那个加载界面可以不要了

**Reply by Minemetero** [reply]
你看看push完了

**Reply by Minemetero** [reply]
UI的话。。。一言难尽

**Reply by lingbopro** [reply]
我要改一下

**Reply by Minemetero** [reply]
要不要给他搞个开源证书

**Reply by lingbopro** [reply]
那得问问 @sjt-2024

**Reply by Minemetero** [reply]
> 我要改一下

你要改什么告诉我

**Reply by lingbopro** [reply]
我push了

**Reply by Minemetero** [reply]
刚玩完战地5，三把连胜ahhhhhh

**Reply by JulyGlow** [reply]
我终于回来了(去吃酒席了
能跟我讲讲刚才发生了什么吗(就是继续干什么

**Reply by Minemetero** [reply]
我去玩战地5了

**Reply by lingbopro** [reply]
我去看B站了

**Comment by JulyGlow** [comment]
话说,你们的IDE背景是什么啊?

**Reply by lingbopro** [reply]
配色方案吗？我用的 One Dark Pro 主题的子主题 One Dark Pro Flat

**Reply by JulyGlow** [reply]
背景啊!
我也是One Dark Pro

**Reply by lingbopro** [reply]
![owo](https://github.com/user-attachments/assets/69b46550-dd72-4553-952d-dad535c2ae4c)

平平无奇的小VSCode

**Reply by JulyGlow** [reply]
你屏幕那么大啊?!
感觉HTML弯弯曲曲的,python还直一点

**Reply by lingbopro** [reply]
我家有个3840x2560的大屏

**Reply by lingbopro** [reply]
> 你屏幕那么大啊?! 感觉HTML弯弯曲曲的,python还直一点

弯弯曲曲？

**Reply by Minemetero** [reply]
> 你屏幕那么大啊?! 感觉HTML弯弯曲曲的,python还直一点

超文本都是这样page要包住里面的东西啊

**Comment by lingbopro** [comment]
下一步我们要干什么？

@sjt-2024 内个，图片…搞到了吗…

**Reply by JulyGlow** [reply]
额...很困难啊

**Reply by Minemetero** [reply]
em?

**Reply by lingbopro** [reply]
嗯…

**Reply by JulyGlow** [reply]
打一堆电话,打不通,还有换了号码的 (真的服了我爸妈

**Reply by lingbopro** [reply]
悲

**Comment by Minemetero** [comment]
我觉得可以搞个文件，把文案什么的集中放，就比如说Title=Hello,然后我HTML里面调用这个Title然后显示
这样子的话，要改文案啥的会方便

**Reply by lingbopro** [reply]
本来就是在`data.js`里的（除了标题），但是页面标题最好还是写在HTML`<title>`里，方便搜索引擎索引

**Comment by Minemetero** [comment]
![image_2024-08-14_102659410](https://github.com/user-attachments/assets/064ef1bc-6e39-4a3f-a6c3-17836e976362)
背景图：我没有招惹你们任何人

**Reply by lingbopro** [reply]
6

**Reply by JulyGlow** [reply]
有毒...

**Reply by JulyGlow** [reply]
真的感觉自己挺傻的...
笨蛋一个...

**Comment by lingbopro** [comment]
![Screenshot_20240814_143819_com.github.android.jpg](https://github.com/user-attachments/assets/3dbb5beb-ee98-4efe-b83b-b1fbb77eb715)

我感觉我天生bug体质…

**Reply by JulyGlow** [reply]
*能跑就行*

**Reply by lingbopro** [reply]
要是我也能有这么热闹的repo就好了…

**Reply by JulyGlow** [reply]
多拉几个协作者,天天肝,你也可以

**Reply by lingbopro** [reply]
我决定下一个项目搞Electron

**Reply by Minemetero** [reply]
学习能力太强了 =.=

**Reply by lingbopro** [reply]
Electron开发的第一个大难关：完整、无错误且快速的安装依赖项（

**Reply by Minemetero** [reply]
理论上是nmp install

**Reply by Minemetero** [reply]
:P

**Reply by lingbopro** [reply]
> 理论上是nmp install

虽然但是，是`npm`不是`nmp`
而且我用`yarn`
并且安装依赖项是在`npx electron-forge import`期间进行的

**Reply by lingbopro** [reply]
我现在在学React

**Reply by lingbopro** [reply]
@Minemetero 我给Sober提了第一个PR！

apprat/sober#10

**Reply by Minemetero** [reply]
这个就这么不了了之了?

**Reply by lingbopro** [reply]
嗯…额…主要是没图

**Reply by JulyGlow** [reply]
还能怎样呢?自学计算机被同学说是装,被爸妈说不务正业,我还能怎样?
And,初二只会压力更大,没人会关注.
20号我就开始失踪,真的没法搞了,不见得国庆假会上线,也不知道我有没有寒假...

**Comment by lingbopro** [comment]
悲伤

![Screenshot_20240818_155623_com.zoho.mail.jpg](https://github.com/user-attachments/assets/4a484bf8-2b96-4116-9b25-2683285b7c9e)

<sup>@Minemetero </sup>

**Reply by Minemetero** [reply]
这是谁。。。

**Reply by Minemetero** [reply]
SJT是什么

**Reply by Minemetero** [reply]
我看出你伤心可以我不能同情。。。

**Reply by JulyGlow** [reply]
@Minemetero SJT是我啊......

**Reply by Minemetero** [reply]
oh我不是人

**Reply by Minemetero** [reply]
我比你们所有人都伤心

**Reply by Minemetero** [reply]
Minemetero@outlook.com

**Reply by Minemetero** [reply]
:)

**Reply by Minemetero** [reply]
云端我没看到你和他的合作啊，除了README[ThromamiomAdd](https://github.com/ThromamiomAdd)

**Reply by lingbopro** [reply]
啊…

**Reply by Minemetero** [reply]
啊什么

**Reply by Minemetero** [reply]
我桑心

**Reply by lingbopro** [reply]
+1

**Reply by Minemetero** [reply]
当不把我当人 :((((((((((((

**Reply by Minemetero** [reply]
转发都不舍得写我名字

**Reply by Minemetero** [reply]
😭

**Reply by lingbopro** [reply]
嗯…我本来想抄送给你的

**Reply by JulyGlow** [reply]
额,突然不知道要说什么了

**Reply by Minemetero** [reply]
我招你惹你了 😭 55555

**Reply by lingbopro** [reply]
可能是因为他不知道你邮箱？

**Comment by Minemetero** [comment]
设成模板开源吧

**Reply by Minemetero** [reply]
这是最后的价值

**Reply by JulyGlow** [reply]
我们可以寒假继续干啊

**Reply by Minemetero** [reply]
?现在就可以干啊，你有什么需求

**Reply by JulyGlow** [reply]
明天我就失踪,我现在听得进Tacitly,但是想不出能干啥.大脑宕机.
嗯...5个月啊...

**Comment by Minemetero** [comment]
把开发者改成模板开发者

**Reply by Minemetero** [reply]
要不开个新repo放

**Reply by lingbopro** [reply]
🤔如果做成模板的话，新repo当然了

**Comment by lingbopro** [comment]
<details>
<summary>展开图片</summary>
<img src="https://github.com/user-attachments/assets/612df803-9582-49f9-b46f-3ec7c9e45ff5" width="314" />
</details>

我透露的信息疑似有点多了（（

**Reply by Minemetero** [reply]
本来不知道 你这样一发全知道了

**Reply by Minemetero** [reply]
![image_2024-08-28_090123759](https://github.com/user-attachments/assets/1b967d0c-7482-4deb-8498-f9bba2c5b2a8)
目标锁定，所有系统全部启动！启动！启动！启动！还有——这个！

**Reply by Minemetero** [reply]
:)

**Reply by lingbopro** [reply]
6

**Comment by lingbopro** [comment]
说起来我想用Electron+React做一个Minecraft启动器，成为业界第二人:P

**Reply by Minemetero** [reply]
为什么是第二？

**Reply by Minemetero** [reply]
MC 启动器你就算了

**Reply by Minemetero** [reply]
卷不过别人的

**Reply by Minemetero** [reply]
除非你拿出极致的轻量，无极的兼容

**Reply by Minemetero** [reply]
不然你就是要给人非凡的体验 超越BakaXL的那种

**Reply by lingbopro** [reply]
> 为什么是第二？

因为已经有人做过基于Electron的了，我记得好像叫XMCL

我指的是基于Electron的MC启动器第二人

**Reply by lingbopro** [reply]
我倒不是想卷，就是想做做，而且有人写过很详细的说明

而且自己的启动器想咋做咋做

**Reply by lingbopro** [reply]
名字我都想好了

**Reply by Minemetero** [reply]
就是照着别人的做一遍。。。

**Reply by lingbopro** [reply]
也不能这么说

**Reply by lingbopro** [reply]
但是毕竟启动方法是一样的

**Reply by lingbopro** [reply]
目前进度：用脚手架创建 React 应用，Electron Forge 导入失败

**Reply by Minemetero** [reply]
你要拿出吸引人的地方啊

**Reply by lingbopro** [reply]
我做这个的主要原因是：
我需要跨平台，HMCL我觉得不太好用，Wine+PCL不舒服，像COLORMC这种UI不太喜欢（（（

**Reply by Minemetero** [reply]
> 我做这个的主要原因是： 我需要跨平台，HMCL我觉得不太好用，Wine+PCL不舒服，像COLORMC这种UI不太喜欢（（（

你看看BakaXL 4.0 你看看 lunar 你看看 LabyModLauncher 再看看什么Badlion

**Reply by Minemetero** [reply]
卷不过啊 你的需求早就被人满足了

**Reply by Minemetero** [reply]
都在搞那种植入性的启动器

**Reply by Minemetero** [reply]
你不做到超越不可能立足啊 但是我举得例子都是大佬级别的存在

**Reply by Minemetero** [reply]
换件事情干干吧，启动器这个领域先放放，还没这个本事

**Reply by Minemetero** [reply]
我想了想 你把你的开发路线图给我 我看看

**Reply by lingbopro** [reply]
首先我本来就不想卷

其次我也没说我要超越什么的，就是做做而已，顺便还能积累点经验什么的

而且啊，自己做的启动器自己还是用着顺手啊:)

**Reply by lingbopro** [reply]
https://xphost008.github.io/lmaml

这个教程真的很详细了

**Reply by Minemetero** [reply]
做了就要做好 不然真没啥意义

**Reply by lingbopro** [reply]
![image](https://github.com/user-attachments/assets/e3ab2662-5ff1-45c7-8abd-0a46f851208e)

超逊的UI（

**Reply by Minemetero** [reply]
So(ber)里So(ber)气

**Reply by lingbopro** [reply]
> So(ber)里So(ber)气

6

**Reply by lingbopro** [reply]
![image](https://github.com/user-attachments/assets/3099cd69-bb28-4112-a9e8-499f9a8601a8)

![image](https://github.com/user-attachments/assets/e0f500d2-e87b-40b3-adb6-ef2409b6570c)

不是很好看的图标（（

**Reply by Minemetero** [reply]
Ai.....解释一下

**Reply by lingbopro** [reply]
？

**Comment by lingbopro** [comment]
![image](https://github.com/user-attachments/assets/9418cb73-9661-4ce8-91f8-199f11bd9adf)

啊啊啊啊啊啊啊Edge我恨你！（bushi

**Comment by lingbopro** [comment]
一早上起来收件箱和邮箱都炸了

<details>
<summary>图片（特长，谨慎打开）</summary>

![Screenshot_20240831_071902_com.github.android.jpg](https://github.com/user-attachments/assets/7d5240eb-d0da-482a-93d0-9b1b44041a64)

</details>

**Reply by lingbopro** [reply]
![Screenshot_20240831_073541.jpg](https://github.com/user-attachments/assets/8b6410ae-bb2d-4dd5-8306-175dca3d36eb)

邮箱文件夹承受了它不该承受的邮件量

**Reply by Minemetero** [reply]
lol

**Comment by lingbopro** [comment]
# 喜！！报！！

![image](https://github.com/user-attachments/assets/2fda383d-4bfd-465a-ad3b-2dab011e4352)

@Minemetero 大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大大喜报！！！！

### @sjt-2024 又上线了！！！！！

**Reply by Minemetero** [reply]
en?又不把我当人

**Reply by lingbopro** [reply]
> en?又不把我当人

我抄送给你

**Reply by JulyGlow** [reply]
# 国庆等我啊!!!
之后就是寒假了.你们放心,虽然每天忙死,我依旧过的很开心(我歌单127首歌,够我熬过这学期)

**Reply by lingbopro** [reply]
> # 国庆等我啊!!!
> 之后就是寒假了.你们放心,虽然每天忙死,我依旧过的很开心(我歌单127首歌,够我熬过这学期)

## 会的，一定会的 :P

**Reply by JulyGlow** [reply]
自从开学,我就没开过VS Code,天天看B站.我一上线就看@Tacitly_Official,所以你们知道我最近在干嘛.
我B站号是@零度爆破者,可以看看.

**Reply by Minemetero** [reply]
看到了

**Reply by lingbopro** [reply]
> 自从开学,我就没开过VS Code,天天看B站.我一上线就看@Tacitly_Official,所以你们知道我最近在干嘛. 我B站号是@零度爆破者,可以看看.

oh?去关注一下

**Reply by lingbopro** [reply]
![Screenshot_20240917_190717_com.orz12.PiliPalaX.jpg](https://github.com/user-attachments/assets/886adffa-5e83-4a3b-801b-4addd0448206)

现在我们三个都有对方的B站号了:D

**Reply by Minemetero** [reply]
:)

**Comment by lingbopro** [comment]
![image](https://github.com/user-attachments/assets/e446529b-ef60-4667-9a07-c2613f08d788)

SJT!!!!!

**Reply by Minemetero** [reply]
更行模组>:(我脑袋要长草了

**Reply by lingbopro** [reply]
> [!NOTE]  
> 我目前不打算更（）

**Reply by JulyGlow** [reply]
咳咳.......本SJT正式上线!

**Reply by lingbopro** [reply]
> 咳咳.......本SJT正式上线!

OHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH!
YEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE!

**Reply by lingbopro** [reply]
Minemetero 那现在是凌晨0:30左右（应该）

**Comment by lingbopro** [comment]
SJT疑似复活

https://t.bilibili.com/1024524333468352512

@Minemetero

**Reply by JulyGlow** [reply]
放心,到点自动复活.15号就复活了 : )

**Reply by lingbopro** [reply]
> 放心,到点自动复活.15号就复活了 : )

:)

**Reply by Minemetero** [reply]
:O

**Reply by kjmjh** [reply]
|ʘᗝʘ|

**Reply by lingbopro** [reply]
> |ʘᗝʘ|

？

**Reply by Minemetero** [reply]
lol

**Comment by JulyGlow** [comment]
啊!咱网站啥时候挂了?
![image](https://github.com/user-attachments/assets/a10fd6e9-f9be-4315-889e-6d99495de49c)

**Reply by lingbopro** [reply]
sober干的.jpg

**Comment by JulyGlow** [comment]
咱仓库都长草了,模版是最后出路,各位怎么看?(或许不用看)

**Reply by Minemetero** [reply]
en同意这个提议

**Reply by lingbopro** [reply]
我看彳亍

**Reply by JulyGlow** [reply]
那,Sober的问题咋搞啊?

**Reply by Minemetero** [reply]
什么sober问题

**Reply by JulyGlow** [reply]
> 什么sober问题  

咱网站早就挂了,往上爬,一月份就说了

**Reply by lingbopro** [reply]
包是sober删版本了

**Reply by lingbopro** [reply]
sober这玩意的早期版本一堆破事

<sub>（刚被学校的神金外包做的脑残作业提交平台整红温了</sub>

**Reply by Minemetero** [reply]
ohh 问题不大，修一修就好了
